Jenkinsfile

```diff
@@ -365,7 +365,9 @@ pipeline {
                       dir("${BASE_DIR}"){
                         powershell label: 'Install test tools', script: '.ci\\windows\\test-tools.ps1'
                         withAzureCredentials(path: "${HOME}", credentialsFile: '.credentials.json') {
-                          bat label: 'Test & coverage', script: '.ci/windows/test.bat'
+                          withTerraformEnv(version: '0.15.3') {
```

Comment:
Seems to work 

<img width="2267" alt="image" src="https://user-images.githubusercontent.com/2871786/169061868-832abd02-4ddd-4586-8d2b-39c013848635.png">